src/redux/reducers/viewerReducer.js

```diff
@@ -118,7 +118,7 @@ export default initialState => (state = initialState, action) => {
             title: payload.tooltip,
             group: payload.buttonGroup,
             img: payload.buttonImage,
-            showColor: 'active',
+            showColor: payload.showColor || 'active',
```

Comment:
we are also using `active` in another changed file;  `if (showColor === 'always' || (showColor === 'active' && isActive)) {`.

should we have a constant for these?

Reply:
I think we will use more constants in this projects at a point, maybe after the new UI is merged.
For now I'm ok with having this just to be a string. It's not inconsistent with other code.